nixos/modules/services/web-apps/zabbix.nix

```
@@ -143,13 +145,25 @@ in
         '';
       };
 
+      extraConfig = mkOption {
+        type = types.lines;
+        default = "";
+        description = ''
+          Additional configuration to be copied verbatim into <filename>zabbix.conf.php</filename>.
+        '';
+      };
+
     };
   };
 
   # implementation
 
   config = mkIf cfg.enable {
 
+    services.zabbixWeb.extraConfig = optionalString ((config.system.stateVersion "20.09") && (versionAtLeast cfg.package.version "5.0.0")) ''
```

It looks like an operator is missing
```suggestion
    services.zabbixWeb.extraConfig = optionalString ((versionAtLeast config.system.stateVersion "20.09") && (versionAtLeast cfg.package.version "5.0.0")) ''
```

---

Thanks for catching that... need to be more careful with my pasting :smile: